.claude/commands/analyze-usage.md

```
@@ -0,0 +1,218 @@
+---
+argument-hint: <project-id> <timeframe>
+description: Analyze BigQuery usage and costs for a project
+---
+
+## Name
+bigquery:analyze-usage
+
+## Synopsis
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)